index.html

```diff
@@ -827,8 +827,36 @@ <h3>
       </li>
       <li>
         <p>
-          Instruct the [= ICE agent =] to use |candidatePair| to send data, and continue to the steps for a change in the selected
-          candidate pair (leading up to {{RTCIceTransport/onselectedcandidatepairchange}}).
+          Let |p:Promise| be a new promise.
+        </p>
+      </li>
+      <li>
+        <p>
+          In parallel, instruct the [= ICE agent =] to use |candidatePair| to send data.
+        </p>
+        <ol>
+          <li>
+            <p>
+              When the [= ICE agent =] has completed selecting |candidatePair|, [= queue a task =] to run the following steps:
+            </p>
+            <ol>
+              <li>
+                <p>
+                  Run the steps for a change in the selected candidate pair for an {{RTCIceTransport}}, leading up to a change in the {{RTCIceTransport/[[SelectedCandidatePair]]}} internal slot and {{RTCIceTransport/onselectedcandidatepairchange}}.
```

Comment:
It should be made clear that the slot change and firing the event are synchronously done.
A follow-up PR should probably make that clear.

Reply:
Correct, once w3c/webrtc-pc#2905 is merged, step 8.1.1 of `selectCandidatePair` will synchronously call `[change the selected candidate pair and state]` algorithm, so that the slot change, firing of the event, and promise resolution is all synchronous.